### PR TITLE
Solving Issue #3155

### PR DESCRIPTION
**Pull request by j0rt3g4**
Solved issue #3155

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-channels-overview.md [hunk 1]
```diff
@@ -106,3 +106,14 @@ It's possible to create Teams that align with the organizational structure. This
 ## Org-wide teams
 
 If your organization has no more than 5,000 users, you can create an org-wide team. Org-wide teams provide an automatic way for everyone in an organization to be a part of a single team for collaboration. For more information, including best practices for creating and managing an org-wide team, see [Create an org-wide team in Microsoft Teams](create-an-org-wide-team.md).
+
+
+# How to hide a channel
+In case that you want to hide a channel inside a teams, right click and select "hide" as shown in the next picture:
```
Comment (JohanFreelancer9) [inline Teams/teams-channels-overview.md]:
```suggestion
If you want to hide a channel inside a team, right click and select **Hide** as shown in the following illustration.
```

File: Teams/teams-channels-overview.md [hunk 2]
```diff
@@ -106,3 +106,14 @@ It's possible to create Teams that align with the organizational structure. This
...
+
+  > ![How to leave a Team](media/HideAChannel.png)
```
Comment (JohanFreelancer9) [inline Teams/teams-channels-overview.md]:
```suggestion
  > ![How to leave a team](media/HideAChannel.png)
```

File: Teams/teams-channels-overview.md [hunk 3]
```diff
@@ -106,3 +106,14 @@ It's possible to create Teams that align with the organizational structure. This
...
+
+# How to leave a Teams
```
Comment (JohanFreelancer9) [inline Teams/teams-channels-overview.md]:
```suggestion
# How to leave a team
```

File: Teams/teams-channels-overview.md [hunk 4]
```diff
@@ -106,3 +106,14 @@ It's possible to create Teams that align with the organizational structure. This
...
+In case that you are willing to get out of a teams, you can right click the channel and then select "Leave the Team" as shown in the next picture:
```
Comment (JohanFreelancer9) [inline Teams/teams-channels-overview.md]:
```suggestion
If you want to leave a team, right click the channel and then select **Leave the Team** as shown in the following illustration.
```

File: Teams/teams-channels-overview.md [hunk 5]
```diff
@@ -106,3 +106,14 @@ It's possible to create Teams that align with the organizational structure. This
...
+ 
+  > ![How to leave a Team](media/LeaveATeam.png)
```
Comment (JohanFreelancer9) [inline Teams/teams-channels-overview.md]: The link to this graphic does not seem to work.

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LolaJacobsen** [comment]
I'm not merging this one. The Teams admin docset isn't intended to provide end user Help. I added a link to https://support.office.com/article/teams-and-channels-df38ae23-8f85-46d3-b071-cb11b9de5499.

**Comment by j0rt3g4** [comment]
Thank you @LolaJacobsen

**Pull request closed without merging**